Spigot-API-Patches/0126-Extend-player-profile-API-to-support-skin-changes.patch

```diff
@@ -0,0 +1,29 @@
+From c54292b8620985a83368e984581d4835604512e1 Mon Sep 17 00:00:00 2001
+From: NickAcPT <32451103+NickAcPT@users.noreply.github.com>
+Date: Sat, 21 Jul 2018 01:30:41 +0100
+Subject: [PATCH] Extend player profile API to support skin changes
+
+Added code that refreshes the player's skin by sending packets with a special order, telling the client to respawn the player and re-apply the game profile
+
+diff --git a/src/main/java/com/destroystokyo/paper/profile/PlayerProfile.java b/src/main/java/com/destroystokyo/paper/profile/PlayerProfile.java
+index e060c38a..98eade06 100644
+--- a/src/main/java/com/destroystokyo/paper/profile/PlayerProfile.java
++++ b/src/main/java/com/destroystokyo/paper/profile/PlayerProfile.java
+@@ -132,6 +132,14 @@ public interface PlayerProfile {
+      * @return If the profile is now complete (has UUID and Name) (if you get rate limited, this operation may fail)
+      */
+     boolean complete(boolean textures);
++    /**
```

Comment:
Based on what the impl is below, i'm not a fan of adding this. Servers should have their desired online mode set correctly using the Paper Bungee Online mode setting.

Reply:
Plus the javadocs are a copy and don't represent what its for.